runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -453,6 +487,61 @@ public void focusLost(FocusEvent e)
 		};
 	}
 
+	private boolean isBuyableSkill(final Skill skill)
+	{
+		switch (skill)
```

Comment:
Don't have a separate body for each case, stack cases which have the same body. Eg:

```java
switch (skill)
{
	case ATTACK:
	case STRENGTH:
	case DEFENCE:
		return false;
	case CONSTRUCTION:
	case HERBLORE:
	case SMITHING:
		return true;
}
```